lib/chorde/_version.py

```diff
@@ -1 +1 @@
-__version__ = '0.8.4'
\ No newline at end of file
+__version__ = '0.9.0'
```

Comment:
Why not release the `1.0.0` version?

Reply:
This is a "migrate to py3" version, with not many functional changes. I feel a 1.0.0 release should have some significant functional changes to merit the "promotion".

Reply:
Deprecate py2 i think that merit the promotion.